python/ray/autoscaler/commands.py

```diff
@@ -448,7 +448,12 @@ def rsync(config_file, source, target, override_cluster_name, down):
             rsync = updater.rsync_down
         else:
             rsync = updater.rsync_up
-        rsync(source, target, check_error=False)
+
+        if source and target:
+            rsync(source, target, check_error=False)
+        else:
```

Comment:
Shouldn't it throw an error if only 1 of source and target is specified?

Reply:
OK, cleaned it up.